antarest/core/utils/fastapi_sqlalchemy/middleware.py

```diff
@@ -16,6 +16,24 @@
 _session: ContextVar[Optional[Session]] = ContextVar("_session", default=None)
 
 
+def _is_sqlite_connection(dbapi_connection: Any) -> bool:
+    cls = dbapi_connection.__class__
+    full_classe_name = f"{cls.__module__}{cls.__name__}"
+    return "sqlite" in full_classe_name.lower()
```

Comment:
I imagine there's not a way to do this in a cleaner way (check of class instance) :/

Reply:
Check of class instance will be more fragile, this is kind of an implementation detail of sqlalchemy driver, I prefer being robust to any implementation change.
It's unlikely that another driver will have sqlite in its name.

Reply:
Ok I see

Reply:
I changed the logic to depend on the driver name instead, it should be a little  more reliable than class names.